custom/icds_reports/utils/aggregation_helpers/distributed/inactive_awws.py

```diff
@@ -107,4 +115,5 @@ def aggregate_query(self):
             table_name=self.aggregate_parent_table,
             ucr_table_query=ucr_query,
             awc_location_table_name='awc_location_local',
+            now=datetime.datetime.utcnow()
```

Comment:
you can directly pass date here instead of passing time and converting it to date in the query.